### PR TITLE
Update modes.dta

### PR DESCRIPTION
**Pull request by glitchgod0**
i had to reclone to change a bool.

### CODE REVIEW

File: _ark/dx/overshell/dx_advanced_states.dta [hunk 1]
```diff
@@ -900,6 +901,12 @@
             {dx_state_pos_tracker}
             DX_DEBUG_DATA
             {set $previous_state dxState_Debug}
+         )
+                  (os_cycleaspect
```
Comment (DarkRTA) [inline _ark/dx/overshell/dx_advanced_states.dta]: fix your indentation here

**Review by DarkRTA — CHANGES_REQUESTED** [review]

**Comment by jnackmclain** [comment]
wait does this not cause unintended behavior? i worry about fully hard setting something in modes

**Comment by glitchgod0** [comment]
I literally couldn't find any issues, I wanna do more testing though

File: _ark/dx/overshell/dx_advanced_states.dta (hunk 1 above)
Comment (glitchgod0) [reply _ark/dx/overshell/dx_advanced_states.dta]: fixed

**Review by DarkRTA — DISMISSED** [review]

**Pull request merged**